### ISSUE TITLE
macports version

### ISSUE THREAD

**Issue by colindean** [badges/shields#9588]
### 📋 Description

I'd like to see a Version category badge for macports using its search interface at https://ports.macports.org.

### 🔗 Data

https://ports.macports.org/api/v1/ports/:package_name

It's achievable with a Dynamic JSON badge for now:

```
https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fports.macports.org%2Fapi%2Fv1%2Fports%2Fproxy-audio-device%2F&query=%24.version&label=macports
```

![proxy-audio-device](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fports.macports.org%2Fapi%2Fv1%2Fports%2Fproxy-audio-device%2F&query=%24.version&label=macports)

### 🎤 Motivation

Shields has badges for several other package managers, and although I'm a fervent Homebrew user, I'd like to see macports supported, too.

**Comment by manzil-infinity180** [comment]
> ### 📋 Description
> I'd like to see a Version category badge for macports using its search interface at https://ports.macports.org.
> 
> ### 🔗 Data
> https://ports.macports.org/api/v1/ports/:package_name
> 
> It's achievable with a Dynamic JSON badge for now:
> 
> ```
> https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fports.macports.org%2Fapi%2Fv1%2Fports%2Fproxy-audio-device%2F&query=%24.version&label=macports
> ```
> 
> ![proxy-audio-device](https://camo.githubusercontent.com/5ec9bf77dfd283ba962ae43232b7bd1102613754c5b4c5116239e0131d3da439/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f64796e616d69632f6a736f6e3f75726c3d6874747073253341253246253246706f7274732e6d6163706f7274732e6f72672532466170692532467631253246706f72747325324670726f78792d617564696f2d6465766963652532462671756572793d2532342e76657273696f6e266c6162656c3d6d6163706f727473)
> 
> ### 🎤 Motivation
> Shields has badges for several other package managers, and although I'm a fervent Homebrew user, I'd like to see macports supported, too.

<img width="1680" alt="Screenshot 2023-09-21 at 12 50 49 AM" src="https://github.com/badges/shields/assets/119070053/99fafad3-40a3-4646-a1ec-00d67476a049">


Provide the url and query i will work on it

**Comment by colindean** [comment]
The URL is `https://ports.macports.org/api/v1/ports/:package_name` where `:package_name` is the package name and the query is simply `$.version`.

**Comment by manzil-infinity180** [comment]
> The URL is `https://ports.macports.org/api/v1/ports/:package_name` where `:package_name` is the package name and the query is simply `$.version`.

OK I will work on that issue under hacktoberfest

**Comment by manzil-infinity180** [comment]
@
<img width="1642" alt="Screenshot 2023-10-01 at 12 57 08 AM" src="https://github.com/badges/shields/assets/119070053/4da82b55-94df-494b-ae26-0e2ea97eab56">
<img width="1642" alt="Screenshot 2023-10-01 at 12 57 04 AM" src="https://github.com/badges/shields/assets/119070053/e181986e-5084-412c-8907-766ba2d343be">

colindean

**Comment by colindean** [comment]
Yes, that's just like the example using the Dynamic JSON that I came up with in my original post! I'd love to see it be a part of the collection of services and not just a Dynamic JSON implementation.

**Comment by manzil-infinity180** [comment]
> Yes, that's just like the example using the Dynamic JSON that I came up with in my original post! I'd love to see it be a part of the collection of services and not just a Dynamic JSON implementation.

can i raise the pr for that

**Comment by colindean** [comment]
Go for it!

**Comment by manzil-infinity180** [comment]
> Go for it!

Merge the pr

**Comment by colindean** [comment]
I do not have merge permissions but the work that you did in that PR is not what the intended work was to be. The intention is to add Macports as a fully-qualified additional service just like all of the others, not using a dynamic Json URL.

**Comment by jcs090218** [comment]
Any update on this?

**Comment by subhan-f** [comment]
Any update on this? I would love to work on this
